docs/.gitignore

```diff
@@ -0,0 +1 @@
+.vitepress
```

Comment:
https://vitepress.dev/guide/getting-started

According to the official documentation we should add `.vitepress/cache` and `.vitepress/dist`. There are useful files inside `.vitepress` that are being excluded by this.

![image](https://github.com/CaiJimmy/stack-docs/assets/5889006/8a9a4ca5-4593-4bce-a080-c422c2a4f64d)

But anyway, this PR is a typo fix, so please exclude this change from it.

Reply:
> But anyway, this PR is a typo fix, so please exclude this change from it.

Done.